go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
 	github.com/OneOfOne/xxhash v1.2.8
-	github.com/edgexfoundry/go-mod-bootstrap/v2 v2.1.0
+	github.com/edgexfoundry/go-mod-bootstrap/v2 v2.0.1-dev.28
```

Comment:
@lenny-intel the version tagging of go-mod-bootstrap might be wrong

Reply:
@cloudxxx8 , it is correct because the tagging of go-mod-bootstrap was setting incorrect after Jakarta release. It has been corrected now and the next tag will be a  `v2.2.0-dev.x`